### PR TITLE
Add `Fake Appearance`

### PR DESCRIPTION
**Pull request by Vladislav-CS**

### CODE REVIEW

File: Exiled.Events/Handlers/Internal/Round.cs [hunk 1]
```diff
@@ -64,6 +65,9 @@ public static void OnChangingRole(ChangingRoleEventArgs ev)
         {
             if (!ev.Player.IsHost && ev.NewRole == RoleTypeId.Spectator && ev.Reason != API.Enums.SpawnReason.Destroyed && Events.Instance.Config.ShouldDropInventory)
                 ev.Player.Inventory.ServerDropEverything();
+
+            if (ev.Player.Role is FpcRole fpcRole)
+                fpcRole.FakeAppearance = null;
```
Comment (louis1706) [inline Exiled.Events/Handlers/Internal/Round.cs:70]: that not needed

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.Events/Handlers/Internal/Round.cs (hunk 1 above)
Comment (louis1706) [inline Exiled.Events/Handlers/Internal/Round.cs:70]:
```suggestion
```

**Review by louis1706 — APPROVED** [review]

File: Exiled.API/Features/Roles/FpcRole.cs [hunk 2]
```diff
@@ -217,5 +219,20 @@ public bool IsNoclipEnabled
...
+        /// </summary>
+        public RoleTypeId? FakeAppearance
+        {
+            get => fakeAppearance;
+            set
+            {
+                fakeAppearance = value;
+
+                if (value.HasValue)
+                    Owner.ChangeAppearance(value.Value);
```
Comment (louis1706) [inline Exiled.API/Features/Roles/FpcRole.cs:234]: How do you reset the value ?

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Review by louis1706 — APPROVED** [review]

**Comment by Misaka-ZeroTwo** [comment]
@Vladislav-CS tested?

**Pull request closed without merging**